src/fivetran-destination/src/destination/dml.rs

```diff
@@ -388,6 +407,152 @@ async fn update_file(
     Ok(())
 }
 
+async fn delete_files(
+    schema: &str,
+    table: &Table,
+    file_config: &FileConfig,
+    client: &tokio_postgres::Client,
+    delete_files: &[String],
+) -> Result<(), anyhow::Error> {
+    // TODO(parkmycar): Make sure table exists.
+    // TODO(parkmycar): Retry transient errors.
+
+    // Copy into a temporary table, which we then merge into the destination.
+    let ts = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .context("time went backwards!")?;
+    // TODO(parkmycar): Make sure table name is <= 255 characters.
+    let temp_table_name = format!("fivetran_{}_temp_delete_{}", table.name, ts.as_millis());
```

Comment:
Add a comment here describing the requirements or lack thereof of this table name.

Reply:
Updated this to use our internal `Ident` structs to make sure this is a valid MZ name, which is the only requirement